onnx/test/test_backend_onnxruntime.py

```diff
@@ -69,9 +69,29 @@ def _create_inference_session(model: onnx.ModelProto, device: str):
         raise ValueError(f"Unexpected device {device!r}.")
     try:
         session = ort.InferenceSession(model.SerializeToString(), providers=providers)
+    except ort.capi.onnxruntime_pybind11_state.InvalidArgument as e:
+        if "is under development and support for this is limited" in str(e):
+            # decrease the opset
+            current_opset = min(d.version for d in model.opset_import if d.domain == "")
+            try:
+                new_model = onnx.version_converter.convert_version(
+                    model, current_opset - 1
+                )
+            except Exception as ne:
+                raise RuntimeError(
+                    f"convert_version was not able to downgrade the opset from "
+                    f"{current_opset}, new_opset={current_opset - 1} due to {ne}"
+                    f"Model is:\n\n{onnx.printer.to_text(model)}"
+                ) from e
+            return _create_inference_session(new_model, device)
+        raise RuntimeError(
+            f"Unable to create inference session due to {e}. "
+            f"Model is:\n\n{onnx.printer.to_text(model)}"
+        ) from e
     except Exception as e:
         raise RuntimeError(
-            f"Unable to create inference session. Model is:\n\n{onnx.printer.to_text(model)}"
+            f"Unable to create inference session due to {e}. "
```

Comment:
We don't need to include e in the message because the `from e` part will already include e in the stack.

Reply:
I prefer to keep it. The error message is long. It is easier to read that way.